packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -109,20 +109,20 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
         Terra.validates.element('click');
       });
 
-      it('keyboard navigates down past the last option', () => {
-        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Space']);
-        Terra.validates.element('no down out of bounds');
-      });
+      // it('keyboard navigates down past the last option', () => {
+      //   browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Space']);
+      //   Terra.validates.element('no down out of bounds');
+      // });
```

Comment:
Are these tests no longer needed, and if so can they be deleted?

Reply:
Deleted.